package.json

```diff
@@ -46,8 +46,8 @@
   "devDependencies": {
     "@types/fs-extra": "^5.0.4",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^6.14.0",
-    "fs-extra": "^7.0.0",
+    "@types/node": "^8.10.43",
+    "fs-extra": "^7.0.1",
```

Comment:
We can remove this now that we are using the builtin `fs`

Reply:
we are using this for the tests , so I think we still need it ?

Reply:
Ah!, got it.